app/views/applications/show.html.erb

```diff
@@ -138,9 +109,14 @@
 
   <%= t.body do %>
     <% @application.latest_deploys_by_environment.each do |environment, deployment| %>
+      <% not_on_default_branch = capture do %>
+        <% if !latest_deploy_on_default_branch.include?(environment) %>
```

Comment:
Super nitpick and I know you just shifted it. Rubocop would complain about this in a .rb file: https://rubystyle.guide/#unless-for-negatives

```suggestion
        <% unless latest_deploy_on_default_branch.include?(environment) %>
```